src/components/Demo/Demo.tsx

```diff
@@ -0,0 +1,15 @@
+import "./demo.css";
+import { demoData } from "../../constants/demoData";
+import Feature from "./Feature";
+
+function Demo() {
+  return (
+    <div className="demo-container flex flex-col">
+      {demoData.map((item) => (
+        <Feature key={item.id} id={item.id} />
```

Comment:
suggestion: tnajem testaamel el built-in index variable li mawjoud fl map() 
dhaherli demoData.map( (item, index) => {
<Feature key={index} id={idex} /> 
)) } tekhdemlk

akkeka maash lezmk t7ot el id fl data bidha (puisque el id maandhech sbab li woujoudha apart el indexing)

Reply:
jarabt badaltha ama tkharej fi erreur donc rajaatha